widgets/WebComponents/AvatarIcon/AvatarIcon.bbj

```diff
@@ -0,0 +1,46 @@
+use ::WebKit/widgets/WebComponents/WebComponents.bbj::WebComponents
+use ::WebKit/util/ClientUtil.bbj::ClientUtil
+
+class public AvatarIcon extends WebComponents
+    field private static BBjTopLevelWindow Wnd!
+    field public static BBjNumber ON_AVATAR_INITIAL_CLICK = 1234
+
+    method public AvatarIcon()
+
+    methodend
+
+    method public BBjHtmlView drawAvatarIcon(BBjWindow wnd!,BBjInt height%, BBjInt width%, BBjString name$)
```

Comment:
duplicate code

Reply:
Hi, it's not duplicate, one function takes integer and other take string (width and height)

Reply:
We don't need two methods performing the exact same task and executing basically the same thing. The method with the int parameters converts the ints into strings and executes the exact same code as the method with the string parameters, making it duplicate code

Reply:
ah you're right. Got it. Fixing now. Thanks.